application/conns.go

```diff
@@ -118,12 +123,21 @@ func (cm *connManager) handleNewConn(regManager *cj.RegistrationManager, clientC
 	}
 	originalDst = originalDstIP.String()
 	flowDescription := fmt.Sprintf("%s -> %s ", originalSrc, originalDst)
-	logger = log.New(os.Stdout, "[CONN] "+flowDescription, golog.Ldate|golog.Lmicroseconds)
+	logger := log.New(os.Stdout, "[CONN] "+flowDescription, golog.Ldate|golog.Lmicroseconds)
+
+	asn, err := regManager.GeoIP.ASN(net.ParseIP(clientConn.RemoteAddr().String()))
+	if err != nil {
+		logger.Errorln("failed to get ASN:", err)
```

Comment:
when err is non-nil  we log it but then continue anyways meaning that it is possible for the asn to be an unexpected value, this will probably cause unexpected behavior. Also not all errors are necessary to log here, for example [the MaxMind.NotFound error](https://pkg.go.dev/github.com/oschwald/maxminddb-golang#pkg-constants) happens when a value is not found which happens more often than you might think. 

We probably don't want to just give up on the connection when things err on geoip lookups, so we may want to add some function (like `isValidASN(asn)`) that we check before updating the stats values. 

_(This comment applies to the CC as well)_